data-raw/survey_comps_datawarehouse.R

```diff
@@ -48,3 +48,14 @@ survey_lcomps(info_surveynames)
 #Generate age comps using function below. Option for CAAL in addition to conditional age comps
 #Saves comps and plots for surveys with age data (only WCGBTS and Triennial)
 survey_acomps(info_surveynames, CAAL = TRUE)
+
+# Copy figures
```

Comment:
Eventually could just put in figures instead of copying from now the data-raw folder.